.github/workflows/ci.yml

```diff
@@ -86,3 +90,14 @@ jobs:
           name: coverage-${{ matrix.working-directory }}-python${{ matrix.python-version }}-slow
           path: ${{ matrix.working-directory }}/.coverage
           include-hidden-files: true
+
+  metrics:
+    needs: [ set-project-matrix, fast-checks, slow-tests ]
```

Comment:
not critical, as part of a later sonar discussion too, but do we want with slow-tests or without? 

`with` means we have our total coverage, but that we won't see the coverage on main unless someone approves the slow tests. `without` reports only unit test coverage in the PRs & main.

Reply:
The latest point where you would want all tests to run is straight before the merge. After that, the damage is done if you missed a red test.